shared-helpers/src/scripts/import-new-translations.ts

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable @typescript-eslint/no-var-requires, import/no-unresolved */
+
+// Usage:
+// ts-node --skip-project src/scripts/import-new-translations.ts tl Tagalog.csv
+//
+// Requires an input CSV with three columns: English, Translation, and (optionally) Updated Translation
```

Comment:
What's an example here?
"I'm a string" "I'm a string in a foreign language" "I'm a string in a foreign language updated"?
I'm not clear what the updated translation would be compared to the translation itself

Reply:
@emily Yeah, I think it's a bit odd too, but that's the format of the spreadsheets/CSVs we got so I had to build that into the script.